README.md

```diff
@@ -89,6 +89,18 @@ Pyright’s type-checking functionality and language features are now incorporat
 Installing both Pyright and Pylance at the same time is not recommended. If both are installed and enabled, you will see duplicate errors, hover text, and completion suggestions. 
 
 
+## Using pyright with pre-commit

```

Comment:
I don't think this documentation belongs in the core README.md file. It should replace the current documentation in the `ci-integration.md` file.

Reply:
In the next commit, it would be updated. 🚀